web/src/compose_closed_ui.js

```diff
@@ -19,7 +19,7 @@ export function get_recipient_label(message) {
     }
 
     if (message === undefined) {
-        if (message_lists.current.visibly_empty()) {
+        if (message_lists.current.data.visibly_empty()) {
```

Comment:
The `class MessageList` implementation in `message_list.js` does have `selected_message` and `visibly_empty`. They just need to be added to the `type MessageList` declaration in `message_lists.ts` (which is temporary, pending the migration of `message_list.js` and its dependencies).

Reply:
Yeah, I did add `selected_message` to  `message_lists.ts`. Should I undo that change, or I keep the change, then when `message_list.js` gets migrated, we remove the change?